src/core/audio/AudioSdl.cpp

```diff
@@ -99,6 +99,9 @@ AudioSdl::AudioSdl( bool & _success_ful, Mixer*  _mixer ) :
 	_success_ful = true;
 
 #ifdef LMMS_HAVE_SDL2
+	// Workaround for a race condition that causes SDL to segfault
+	SDL_Delay(250);
```

Comment:
Is 250ms a really good time interval? I'm not sure, but you can try lowering this and try to reproduce the crash.